features/admin.applications.v1/configs/endpoints.ts

```diff
@@ -29,6 +29,7 @@ export const getApplicationsResourceEndpoints = (serverHost: string): Applicatio
 
     return {
         applications: `${ serverHost }/api/server/v1/applications`,
+        asyncStatus: `${ serverHost }/api/server/v1/async-operations`,
```

Comment:
This should ideally be in admin.core.v1 as this is not an specific endpoint for app mgt